.eleventy.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function (eleventyConfig) {
+  eleventyConfig.addPassthroughCopy("docs/css");
+  eleventyConfig.addPassthroughCopy("docs/js");
+  eleventyConfig.addPassthroughCopy("docs/images");
+  eleventyConfig.addPassthroughCopy("docs/CNAME");
+  eleventyConfig.addPassthroughCopy("docs/_headers");
+  eleventyConfig.addPassthroughCopy("docs/favicon.ico");
+
+  eleventyConfig.addPassthroughCopy("docs/example");
+
+  /* Markdown Plugins */
+  const markdown = require("markdown-it")({
```

Comment:
I just went with `markdown-it` because it is the eleventy default. I'm not specifically drawn to a certain markdown tool. That said, I found the documentation for `markdown-it` and its plugins quite comprehensible, which I find to be a good trait. I'll see if I can find some way to consolidate

Reply:
thanks.  I was loathe to use both `markdown-magic` and `remark` in the first place.  we can probably drop `markdown-magic` and even `markdownlint` for all the stuff `remark` does...